lib/routes/emails.js

```diff
@@ -14,7 +14,9 @@ const validators = require('./validators');
 
 const HEX_STRING = validators.HEX_STRING;
 
-module.exports = (log, db, mailer, config, customs, push) => {
+module.exports = (log, db, mailer, config, customs, push, verificationReminders) => {
+  const REMINDER_PATTERN = new RegExp(`^(?:${verificationReminders.keys.join('|')})$`);
```

Comment:
You could export the regex directly from `verificationReminders` or put into the validators lib.

Reply:
I'm minded to keep this where it is because it's local to this module and conceptually it seems like the right place. Exporting it from `lib/verification-reminders` pollutes that module with knowledge about the API surface and putting it in the validators lib would force that to import `lib/verification-reminders`.